guides/14_Content_Security_Policy.md

```diff
@@ -0,0 +1,38 @@
+# Content Security Policy for applications using the Amazon Chime SDK
+
+Modern web applications use [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to protect users from certain classes of attacks. You can add a security policy by configuring your web server to return the Content-Security-Policy HTTP header or using <meta> element to configure a policy. A guide to use a Lambda@Edge function to inject security headers through CloudFront, see [Adding HTTP Security Headers Using Lambda@Edge and Amazon CloudFront](https://aws.amazon.com/blogs/networking-and-content-delivery/adding-http-security-headers-using-lambdaedge-and-amazon-cloudfront/).
```

Comment:
- `<meta>` does not show up in the markdown viewer. Use a code block:
  ```
  `<meta>`
  ```
- `A guide to ...` is an incomplete sentence. How about the following?

   > You can add security headers using [Lambda@Edge](http://docs.aws.amazon.com/lambda/latest/dg/lambda-edge.html) and [Amazon CloudFront](https://aws.amazon.com/cloudfront/). For more information, see the [Adding HTTP Security Headers Using Lambda@Edge and Amazon CloudFront](https://aws.amazon.com/blogs/networking-and-content-delivery/adding-http-security-headers-using-lambdaedge-and-amazon-cloudfront/).

Reply:
Good point. Will fix!